tracer/build/_build/Projects.cs

```diff
@@ -46,7 +46,8 @@ public static class Projects
         AppSecUnitTests,
         ClrProfilerManagedTests,
         TraceIntegrationTests,
-        TraceTests
+        TraceTests,
+        DdTraceIntegrationTests
```

Comment:
s)this list is going to include everything soon except `Datadog.Trace.Tests` (basic unit tests), right? 😅